src/main/java/com/redhat/devtools/lsp4ij/LSPIJTextMateUtils.java

```diff
@@ -0,0 +1,92 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+
+package com.redhat.devtools.lsp4ij;
+
+import com.intellij.psi.PsiFile;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Map;
+
+/**
+ * Utilities for working with TextMate files in LSP4IJ.
+ */
+@ApiStatus.Internal
+public final class LSPIJTextMateUtils {
+
+    private LSPIJTextMateUtils() {
+        // Pure utility class
+    }
+
+    /**
+     * Returns the simple/single-character brace pairs for the file if it's a TextMate file.
+     *
+     * @param file the PSI file
+     * @return the simple brace pairs for the file if it's a TextMate file; otherwise null
+     */
+    @Nullable
+    @ApiStatus.Internal
+    public static Map<Character, Character> getBracePairs(@NotNull PsiFile file) {
```

Comment:
This method should be removed, right?

Reply:
It's up to you. I'd hate to lose it if LSP4IJ is going to be updated to drop support for 2023.2 soon and it should be restored, and it would likely be lost through a squash merge. If you'd like it removed since it's a no-op in its current form, I can do so, but it should hopefully only be commented out for a short time until 2023.2 support is dropped. Let me know what you'd prefer.